providers/cert.rb

```diff
@@ -19,8 +19,39 @@
         end.run_action(:install)
       end
 
+      # Install PK11 kit if needeed by overwritting Firefox's
+      # libnssckbi.so file. By doing this Firefox will trust
+      # system certificates
+      libnssckbi = ShellUtil.shell(
+        'dpkg -L firefox | grep libnssckbi.so | head -n 1'
+      ).stdout.chomp
+
+      p11_kit_trust = ShellUtil.shell(
+        'dpkg -L p11-kit-modules | grep p11-kit-trust.so | head -n 1'
+      ).stdout.chomp
+
+      if ::File.exist?(libnssckbi) && ::File.exist?(p11_kit_trust) &&
+         ::File.stat(libnssckbi).size != ::File.stat(p11_kit_trust).size
+        Chef::Log.info('Overwrite libnssckbi.so file')
+        ::FileUtils.cp p11_kit_trust, libnssckbi
```

Comment:
What if we just use a symbolic link instead of copying onto the Firefos libnssckbi.so file?

(Much better if we make a debian package diverting that file, or using alternatives)

Reply:
I've just tested it and a symbolic link works fine too.

Diverting that file with another package could be done in GECOS Linux distributions, but if you have plans to allow using Ubuntu Linux distributions in GECOS systems this could be a problem. 

What do you consider to be the best solution?

Best regards!

Reply:
Meanwhile a definitive solution comes up in Ubuntu/Debian, a symbolic link seems more reversible.

Please, change the overwrite to a link so we can test this complex PR ASAP .